services/runModels.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+import pandas as pd
+import pickle
+
+def getRandomForestPredictions(X):
+  rf_model = pickle.load(open('highlight-model/rf_100t.sav', 'rb')) # Get model from file
```

Comment:
nit: maybe we should put the model files in a the datastore folder for consistency?

Reply:
Fixed